web/web.go

```diff
@@ -121,6 +129,7 @@ func ConfigHandler(
 			AlgoliaDocsIndexName:     cfg.Algolia.DocsIndexName,
 			AlgoliaDraftsIndexName:   cfg.Algolia.DraftsIndexName,
 			AlgoliaInternalIndexName: cfg.Algolia.InternalIndexName,
+			CreateDocsAsUser:         createDocsAsUser,
```

Comment:
Curious why we need to send it in the config?

Reply:
I was thinking that we could make the modal that pops up after a draft is created (or at least the text in it about users having to manually manage their notifications) conditional on this.

Reply:
That is a good idea